doc/TOR.md

```diff
@@ -100,6 +185,44 @@ is necessary to set up a [Tor Hidden Service].
 *NOTE: In the case where only non-persistent addresses are required,
 you don't have to create the hidden service and you can skip this part.*
 
+##### Automatic persistent .onion address
+
+It is possible to generate persistent .onion addresses automatically.
+
+Add the following lines in the `/etc/tor/torrc` file
+(you might already have done this if for example you connected Bitcoin
+over Tor):
+
+````
+ControlPort 9051
+CookieAuthentication 1
+CookieAuthFileGroupReadable 1
+````
+
+Then you can use `--addr=statictor:127.0.0.1:9051` instead of
+`--announce-addr=.onionAddressV3`.
+By default V3 onion addresses are generated.
+
+Note that you have to specify a `--bind-addr` first before using
+`--addr=statictor:`.
+Generally `--bind-addr=127.0.0.1:9735` should work fine.
+
+You can also have multiple announced persistent .onion addresses
+by adding `/torblob=BLOB`, where `BLOB` is up to 32 to 64 bytes of
```

Comment:
Wow that protocol change i really missed,
I always wanted to announce multiple address of the same type, but was overruled like multiple regtest in early dev.
Although i guess it was a typo and should word like.
Sugessted change:

> You can defined multiple persistent .onion addresses and Tor will route them from and to your node,
> but lightning in it's BOLTS is a bit strict here and only allows one address of a type, so
> only the the first address of v3 onion type you define will be announced over LN gosssp

Reply:
Oh.... that is sad.  Will fix.